test/interactions/keyInteractionTests.ts

```diff
@@ -116,6 +116,36 @@ describe("Interactions", () => {
         keyInteraction.offKeyPress(aCode, aCallback2);
         svg.remove();
       });
+
+      it("is only triggered once when the key is pressed", () => {
+        keyInteraction.onKeyPress(aCode, aCallback);
+        keyInteraction.attachTo(component);
+
+        TestMethods.triggerFakeMouseEvent("mouseover", component.background(), 100, 100);
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aCode);
+        assert.isTrue(aCallbackCalled, "callback for \"a\" was called when \"a\" key was pressed");
+
+        aCallbackCalled = false;
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aCode, { repeat : true });
+        assert.isFalse(aCallbackCalled, "callback for \"a\" was not called when keydown was fired the second time");
+
+        keyInteraction.offKeyPress(aCode, aCallback);
+        svg.remove();
+      });
+
+      it("does not fire callback pressing the key when mouse was outside and moved inside", () => {
```

Comment:
"does not fire the callback if the key is pressed and held down outside, then the mouse moved inside"